client/package.json

```diff
@@ -49,7 +49,7 @@
     "react-redux": "^7.2.1",
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "^4.0.1",
+    "react-scripts": "^4.0.2",
```

Comment:
As of react-scripts 4.0.2, BUILD_PATH feature is provided.

https://create-react-app.dev/docs/advanced-configuration/